style.css

```diff
@@ -0,0 +1,731 @@
+}
+
+#front-page {
+  background-image: url(./visuals/front-page-background.jpg);
+  background-size: cover;
+  background-position: top;
+  background-attachment: fixed;
+}
+
+#front-page__content {
+  margin: 16.6vh 0;
+}
+
+#front-page__title {
+  mix-blend-mode: color-burn;
+  filter: contrast(120%) saturate(200%);
+}
+
+.support__logo {
+  width: 20rem;
+  filter: saturate(0%) opacity(50%);
+}
+
+.support__logo:hover {
+  filter: saturate(100%) opacity(100%);
+}
+
+/* BAND MEMBERS PAGE */
+
+.band-member-cards__portrait {
+  margin-top: 28px;
+  margin-left: 28px;
+  max-width: 170px;
+  max-height: 170px;
+  box-shadow: -70.84px -70.84px 0 -42.5px;
+}
+
+/* ABOUT PAGE */
+#about {
+  background-image: url(./visuals/about-background.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-attachment: fixed;
+}
+
+#embark__logo {
+  max-width: 33rem;
+  width: 100%;
+}
+
+.older-wt {
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top;
+  filter: saturate(120%);
+  box-shadow: #ec5342b2 0 0 0 500px inset;
+  height: 20rem;
+}
+
+.older-wt:hover {
+  box-shadow: #0000006f 0 0 0 500px inset;
+}
+
+#wt-japan {
+  background-image: url("./visuals/wt-japan.jpg");
+}
+
+#wt-brazil {
+  background-image: url("./visuals/wt-brazil.jpg");
+}
+
+/* Mobile Menu */
+
+#hamburger {
+  display: none;
+}
+
+.bi-x {
+  position: relative;
+  top: 0;
+}
+
+#navbar__hamburguer__menulist {
+  display: none;
+  opacity: 0;
+}
+
+@media only screen and (max-width: 1250px) {
+  .grid {
+    grid-template-columns: 7vw 1fr 7vw;
+  }
+}
+
+@media only screen and (max-width: 1100px) {
+  #navbar__container {
+    gap: 20px;
+  }
+
+  #navbar__container,
+  #dates__container,
+  .band__paired-members,
+  #past-wt__container {
+    flex-direction: column;
+  }
+
+  #front-page__title h2 {
+    font-size: 5rem;
+  }
+
+  #front-page__title__world-tour,
+  #front-page__body-text {
+    max-width: 100%;
+  }
+}
+
+@media only screen and (max-width: 1000px) {
+  .max-w-84 {
+    max-width: 100%;
+  }
+
+  #about h1 {
+    font-size: 7rem;
+  }
+}
+
+@media only screen and (max-width: 850px) {
+  .max-w-75 {
+    max-width: 100%;
+  }
+
+  #navbar__menulist {
+    display: none;
+  }
+
+  #navbar__container {
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 0;
+  }
+
+  nav {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1;
+  }
+
+  #navbar__button-logo {
+    width: 100%;
+  }
+
+  #hamburger {
+    display: inline;
+  }
+
+  #navbar__logo {
+    width: 10rem;
+  }
+
+  #hamburger i {
+    font-size: 3rem;
+  }
+
+  #about h1 {
+    font-size: 5rem;
+  }
+}
+
+@media only screen and (max-width: 700px) {
+  .dates__paired {
+    flex-direction: column;
+  }
+
+  #front-page {
+    background-position: -400px;
+  }
+}
+
+@media only screen and (max-width: 645px) {
+  #front-page {
+    background-position: -600px;
+  }
```

Comment:
Great design @grauJavier 🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the given design as much as possible.
- [ ] Kindly align the text properly so that is appears as the one in the design provided.
- [ ] You may also remove the logo from your mobile view so as to replicate the design given.
- Your design
![Your mobile home](https://github.com/grauJavier/capstone-project/assets/63917725/7da00314-245c-411c-be9d-b31cf8245670)
- Given design
![Given mobile home](https://github.com/grauJavier/capstone-project/assets/63917725/5811dbf6-5a2b-4c2c-b5c6-e7621661adc4)